documentation/style-guide.rst

```diff
@@ -196,6 +199,35 @@ language):
     achieve the same effect.  This assures that files are flushed and file
     descriptor resources are released in a timely manner.
 
+
+Author attribution
+==================
+
+For new documentation, do not use a byline (naming the author of the document).
+Explicit attribution tends to discourage other users from updating community
+documentation.
+
+Existing documentation with bylines will not be changed unless the author
+decides to do so. This is subject to change in the future.
+
+
+First-person language
+=====================
+
+Avoid first-person language.  Our documentation is owned and maintained by a
+community.  First-person language mistakenly implies a single author.
+
+
+Pronunciation of dunders
+========================
```

Comment:
```suggestion
Pronunciation of dunder methods
===============================
```

Reply:
I left out "methods" because there are a few that are not methods.  How about "dunder names"?